### PR TITLE
List BitcoinV (BTCV)

### PR DESCRIPTION
**Pull request by ghost**
Please add BitcoinV to BISQ. 

Block explorer: http://explorer.bitcoinv.org
Website: http://explorer.bitcoinv.org

Thanks, 
NullFunctor support@bitcoinv.org

### CODE REVIEW

**Comment by wiz** [comment]
NACK - the name of this crypto-currency is too similar to Bitcoin and might confuse users.

**Comment by JSDapunta** [comment]
I prefer this coin than Bitcoin

**Comment by ghost** [comment]
> NACK - the name of this crypto-currency is too similar to Bitcoin and might confuse users.

BitcoinV has been listed on Graviex for a long time. There hasn't been any confusion.
https://graviex.net/markets/btcvbtc

**Comment by ghost** [comment]
BitcoinV has a growing list of supporters. Currently we have about 800 followers on Telegram:
https://t.me/BitCoinVcryptocurrency
We are on Graviex and BitParax exchanges. The community wants to grow BitcoinV by becoming a part of the BISQ network.

File: assets/src/main/java/bisq/asset/coins/BitcoinV.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+package bisq.asset.coins;
+
+import bisq.asset.Base58BitcoinAddressValidator;
+import bisq.asset.Coin;
+
+import org.bitcoinj.core.NetworkParameters;
+import org.bitcoinj.params.MainNetParams;
+
+public abstract class BitcoinV extends Coin {
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/BitcoinV.java]: No need for abstraction here. Make this class use mainnet (default params).

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK do not push invalid commits without testing them locally!!!

**Comment by ghost** [comment]
Sorry about the invalid commits. Will test locally from now on.

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - https://docs.bisq.network/exchange/howto/list-asset.html#step-5-create-a-well-formed-git-commit

**Comment by ripcurlx** [comment]
~@bitcoinVBR Do you read comments? https://github.com/bisq-network/bisq/pull/3252#pullrequestreview-311557220~ Sorry, my bad - missed the commit.
But NACK again because of https://help.github.com/en/github/authenticating-to-github/signing-commits

**Comment by ripcurlx** [comment]
But double/triple NACK - as you still not comply with https://docs.bisq.network/exchange/howto/list-asset.html#step-5-create-a-well-formed-git-commit

**Comment by chimp1984** [comment]
I think we made once a decision that coins abusing the Bitcoin brand by using it in their name are not supported. I would vote to remove that coin if it gets it.

**Comment by pianharianto** [comment]
Good project

**Comment by Honeyboy12** [comment]
Bitcoinv is a good project and a great coin,please give it a chance

**Comment by ripcurlx** [comment]
> I think we made once a decision that coins abusing the Bitcoin brand by using it in their name are not supported. I would vote to remove that coin if it gets it.

Not sure if that is documented somewhere, but it can definitely put for a vote after it is integrated.

For me as maintainer as long as you are not able to follow https://docs.bisq.network/exchange/howto/list-asset.html this PR won't be merged.

**Review by JayMedyaa — APPROVED** [review]

**Comment by munasifinv** [comment]
Where to get BitcoinV wallet ?

**Comment by ghost** [comment]
https://github.com/BitcoinVBR/bitcoinV/releases/tag/v0.20.0.1_final

**Pull request closed without merging**